crates/hyperswitch_domain_models/src/payments/payment_intent.rs

```diff
@@ -138,12 +150,13 @@ pub enum PaymentIntentUpdate {
         metadata: pii::SecretSerdeValue,
         updated_by: String,
     },
-    ReturnUrlUpdate {
+    PaymentCreateUpdate {
```

Comment:
What about updating in payments update flow? can customer details be updated there?